wdl-engine/src/eval.rs

```diff
@@ -459,46 +459,64 @@ impl EvaluatedTask {
+            //             let reader = RevBufReader::new(f);
+            //             let lines: Vec<_> = reader
+            //                 .lines()
+            //                 .take(MAX_STDERR_LINES)
+            //                 .map_while(|l| l.ok())
+            //                 .collect();
+
+            //             // Iterate the lines in reverse order as we read them in reverse
+            //             lines
+            //                 .iter()
+            //                 .rev()
+            //                 .format_with("\n", |l, f| f(&format_args!("  {l}")))
+            //                 .to_string()
+            //         })
+            //     })
+            //     .unwrap_or_default();
+
+            let stdout = download_file(downloader, None, self.stdout().as_file().unwrap())
+                .await
+                .ok()
+                .and_then(|l| fs::read_to_string(l).ok())
+                .unwrap_or_default();
+
             let stderr = download_file(downloader, None, self.stderr().as_file().unwrap())
                 .await
                 .ok()
-                .and_then(|l| {
-                    fs::File::open(l).ok().map(|f| {
-                        // Buffer the last N number of lines
-                        let reader = RevBufReader::new(f);
-                        let lines: Vec<_> = reader
-                            .lines()
-                            .take(MAX_STDERR_LINES)
-                            .map_while(|l| l.ok())
-                            .collect();
-
-                        // Iterate the lines in reverse order as we read them in reverse
-                        lines
-                            .iter()
-                            .rev()
-                            .format_with("\n", |l, f| f(&format_args!("  {l}")))
-                            .to_string()
-                    })
-                })
+                .and_then(|l| fs::read_to_string(l).ok())
                 .unwrap_or_default();
 
             // If the work directory is remote,
             bail!(
                 "process terminated with exit code {code}: see `{stdout_path}` and \
                  `{stderr_path}` for task output and the related files in \
-                 `{dir}`{header}{stderr}{trailer}",
+                 `{dir}`{stdout_header}{stdout}{stdout_trailer}{stderr_header}{stderr}{stderr_trailer}",
                 code = self.result.exit_code,
                 dir = self.attempt_dir().display(),
                 stdout_path = self.stdout().as_file().expect("must be file"),
                 stderr_path = self.stderr().as_file().expect("must be file"),
-                header = if stderr.is_empty() {
+                stdout_header = if stderr.is_empty() {
+                    Cow::Borrowed("")
+                } else {
+                    format!("\n\ntask stdout output (last {MAX_STDERR_LINES} lines):\n\n").into()
```

Comment:
If we keep this, this should just be `MAX_LINES`.

Reply:
Oh no, it was just a commit to get some diagnostics out of CI, no intent to keep it.